ci/integration_tests.sh

```diff
@@ -9,45 +9,36 @@ export JRUBY_OPTS="-J-Xmx1g"
 export SPEC_OPTS="--order rand --format documentation"
 export CI=true
 
-rm -rf build && mkdir build
-
-echo "Building tar"
-rake artifact:tar
-cd build
-tar xf *.tar.gz
-
-cd ../qa/integration
-echo "Installing test dependencies"
-bundle install
-
 if [[ $1 = "setup" ]]; then
  echo "Setup only, no tests will be run"
  exit 0
 
 elif [[ $1 == "split" ]]; then
+    cd qa/integration 
     glob1=(specs/*spec.rb)
     glob2=(specs/**/*spec.rb)
     all_specs=("${glob1[@]}" "${glob2[@]}")
 
     specs0=${all_specs[@]::$((${#all_specs[@]} / 2 ))}
     specs1=${all_specs[@]:$((${#all_specs[@]} / 2 ))}
-
+    cd ..
+    cd ..
```

Comment:
nit: one line cd, else imo, it looks like a typo

Reply:
@jakelandis right sorry that was weird :D fixing